app/db/models/index.js

```diff
@@ -365,7 +364,7 @@ module.exports = function (sequelize, env) {
             }
           }
         })
-        .then(function (loginToken) {
+        .then(loginToken => {
           if (!loginToken) {
             return bPromise.reject({
```

Comment:
the problem with this is that sequelize uses an independent copy of `bluebird` so I'm not sure if rejecting a promise like this would conflict with that.